### PR TITLE
Add S3 parquet file - arrow accelerator e2e test

### PR DESCRIPTION
**Pull request by Sevenannn**
## 🗣 Description

- Note: Clickhouse e2e is temporarily disabled due to this bug: https://github.com/spiceai/spiceai/issues/3184

Add S3 parquet file - arrow accelerator / no accelerator e2e test.

The schema of S3 parquet file is as follows. SQLite / DuckDB / Postgres accelerator are not added to test yet since they don't fully support these parquet types
```
+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+
| column_name   | data_type                                                                                                                                                                                                                                                                                                                                               | is_nullable |
+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+
| boolean_col   | Boolean                                                                                                                                                                                                                                                                                                                                                 | YES         |
| int32_col     | Int32                                                                                                                                                                                                                                                                                                                                                   | YES         |
| int64_col     | Int64                                                                                                                                                                                                                                                                                                                                                   | YES         |
| float_col     | Float32                                                                                                                                                                                                                                                                                                                                                 | YES         |
| double_col    | Float64                                                                                                                                                                                                                                                                                                                                                 | YES         |
| string_col    | Utf8                                                                                                                                                                                                                                                                                                                                                    | YES         |
| binary_col    | Binary                                                                                                                                                                                                                                                                                                                                                  | YES         |
| date_col      | Date32                                                                                                                                                                                                                                                                                                                                                  | YES         |
| timestamp_col | Timestamp(Microsecond, None)                                                                                                                                                                                                                                                                                                                            | YES         |
| decimal_col   | Decimal128(18, 2)                                                                                                                                                                                                                                                                                                                                       | YES         |
| list_col      | List(Field { name: "element", data_type: Int32, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} })                                                                                                                                                                                                                                     | YES         |
| map_col       | Map(Field { name: "key_value", data_type: Struct([Field { name: "key", data_type: Utf8, nullable: false, dict_id: 0, dict_is_ordered: false, metadata: {} }, Field { name: "value", data_type: Int32, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }]), nullable: false, dict_id: 0, dict_is_ordered: false, metadata: {} }, false) | YES         |
| struct_col    | Struct([Field { name: "id", data_type: Int32, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }, Field { name: "name", data_type: Utf8, nullable: true, dict_id: 0, dict_is_ordered: false, metadata: {} }])                                                                                                                           | YES         |
+---------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-------------+
```

Successful run: https://github.com/spiceai/spiceai/actions/runs/11413115123/job/31760669101

## 🔨 Related Issues

<!-- list any linked issues this pull request will close, or exclude if none -->

## 📄 Documentation Requirements

<!-- list any documentation related iusses, quickstarts/samples or video content related to this PR -->

## 🤔 Concerns

<!-- list any particular concerns you have about this pull request that you want reviewers to directly address, or exclude if none -->

### CODE REVIEW

File: .github/workflows/e2e_test_ci.yml
Comment (phillipleblanc) [inline .github/workflows/e2e_test_ci.yml]:
```suggestion
          while [[ "$(curl -s http://localhost:8090/v1/ready)" != "ok" ]]; do sleep 1; done
```

Comment (phillipleblanc) [reply .github/workflows/e2e_test_ci.yml]: Should this be the readiness handler instead?

**Review by phillipleblanc — DISMISSED** [review]

File: .github/workflows/e2e_test_ci.yml
Comment (Sevenannn) [reply .github/workflows/e2e_test_ci.yml]: Yes it should, let me update that

File: .github/workflows/e2e_test_ci.yml
Comment (phillipleblanc) [inline .github/workflows/e2e_test_ci.yml]: Let's wait to merge this PR once my clickhouse fix is merged and we can re-enable this.

**Review by peasee — APPROVED** [review]

**Pull request merged**
